### PR TITLE
Remove initial cra stuff, add cards, add theme ui for styling

### PR DESCRIPTION
**Pull request by dbertella**
image
Card rendering is not perfect, I'm using a big sprite and it's not probably cut very well, but look, we have cards!

### CODE REVIEW

File: packages/client/src/cards.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { range } from 'lodash';
+
+export const SUIT = {
+  Denari: 'denari',
```
Comment (stathismor) [inline packages/client/src/cards.tsx]: Can we use the English names for these? Same for numbers.

File: packages/client/src/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  5: 'Cinque',
+  6: 'Sei',
+  7: 'Sette',
+  8: 'Fante',
+  9: 'Cavallo',
+  10: 'Re',
+};
+
+function name([value, suit]: CardType) {
+  return `${VALUES[value]} di ${SUIT[startCase(suit)]}`;
```
Comment (stathismor) [inline packages/client/src/Card.tsx]:
```suggestion
  return `${VALUES[value]} of ${SUIT[startCase(suit)]}`;
```

File: packages/client/src/cards.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { range } from 'lodash';
```
Comment (stathismor) [inline packages/client/src/cards.tsx:1]: Don't know about conventions here, wondering if having a file named `Card` and one called `cards` gets confusing. Maybe `deck`? Don't know, don't care that much.

**Review by stathismor — APPROVED** [review]
I merged so you have conflicts now, sorry :)

File: packages/client/src/cards.tsx (hunk 1 above)
Comment (dbertella) [reply packages/client/src/cards.tsx]: I think those are italian names not sure the play well if we translate. I copied the structure from someone else but I think it's a good idea for this game

File: packages/client/src/cards.tsx (hunk 3 above)
Comment (dbertella) [reply packages/client/src/cards.tsx:1]: Yeah we will think about a better structure later, this is temporary anyway

File: packages/client/src/cards.tsx (hunk 1 above)
Comment (stathismor) [reply packages/client/src/cards.tsx]: But it's just a 1:1 translation from the cards, right? The names are not unique to the game, or unique enough, they are just the card numbers and symbols. You can just take some cards off, from a common deck and play the game.
If we put Italian text like that, I think it's mostly going to be targeting Italians. It is too much to ask from non-Italians to learn/remember.

**Pull request merged**